tests/unittests/gradCheckTest.cpp

```diff
@@ -199,6 +199,74 @@ TEST(Network, gradientCheckConv) {
   performGradCheck(IP, result, A, Ex, &inputs, &outputs, 0.001, 0.004);
 }
 
+TEST(Network, gradientCheckDepthwiseConv) {
+  ExecutionEngine IP;
```

Comment:
Looks like these 2 new tests are very similar to the existing test. Could you please move tests' body to a new function, so that all 3 tests call it? Looks like only group number and number of input channels are changing.